plugins/hls-haddock-comments-plugin/src/Ide/Plugin/HaddockComments.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE RecordWildCards #-}
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# OPTIONS_GHC -Wall #-}
```

Comment:
I think it's better to enable this option for entire library via `hls-haddock-comments-plugin.cabal`.

Reply:
Ah, this pragma was added by accident. There's already one in `hls-haddock-comments-plugin.cabal`.